tools/bin/capture-package-versions

```diff
@@ -2,7 +2,8 @@
 
 set -e -o pipefail
 
+csd=$(dirname $0)
 DOCKER_IMAGE=$1
 OUTPUT_FILE=$2
 
-docker run --rm -ti $DOCKER_IMAGE dpkg-query -W -f "\${package},\${version}\n" | jq -s -R -f /usr/local/bin/csv-to-array.jq > $OUTPUT_FILE
+docker run --rm -ti $DOCKER_IMAGE dpkg-query -W -f "\${package},\${version}\n" | jq -s -R -f ${csd}/csv-to-array.jq > $OUTPUT_FILE
```

Comment:
This works on Travis due to:
https://github.com/heroku/stack-images/blob/fd80801050221a436108308a7a705d7eaf840a1c/bin/convert-and-publish-to-heroku.sh#L3

I'm presuming this script was being run by hand locally? If so, the way we've been updating the installed-packages list until now is via the `bin/build.sh` script, which doesn't use this file. 

There's definitely some additional clean-up that can be performed now that the stack image tools repo has been merged into this one (of which this is one part) - however since it's unrelated to this PR and not necessary to update the packages list locally, I think we should exclude this change for now.

Reply:
I'll remove this commit 👍